File: Stage2/C/Problem2480.c
#include <stdio.h>

int main() {
    int dice1, dice2, dice3, reward;

    scanf("%d %d %d", &dice1, &dice2, &dice3);

    if (dice1 == dice2 && dice2 == dice3) {
        reward = 10000 + dice1 * 1000;
    } else if (dice1 == dice2 || dice2 == dice3 || dice1 == dice3) {
        int sameValue = (dice1 == dice2 || dice1 == dice3) ? dice1 : dice2;
        reward = 1000 + sameValue * 100;
    } else {
        int maxValue = dice1;
        if (dice2 > maxValue) maxValue = dice2;
        if (dice3 > maxValue) maxValue = dice3;
        reward = maxValue * 100;
    }

    printf("%d\n", reward);

    return 0;
}

File: Stage2/Python/Problem2480.py
def main():
    dice1, dice2, dice3 = map(int, input().split())
    
    if dice1 == dice2 and dice2 == dice3:
        reward = 10000 + dice1 * 1000
    elif dice1 == dice2 or dice2 == dice3 or dice1 == dice3:
        sameValue = dice1 if (dice1 == dice2 or dice1 == dice3) else dice2
        reward = 1000 + sameValue * 100
    else:
        maxValue = max(dice1, dice2, dice3)
        reward = maxValue * 100
        
    print(reward)
    
if __name__ == "__main__":
    main()